js/views/userPageVw.js

```diff
@@ -307,6 +307,25 @@ module.exports = baseVw.extend({
           self.model.set({ownPage: self.options.ownPage});
           self.render();
           !self.currentItemHash && self.loadingDeferred.resolve();
+          
+          // Handle was requested
+          if (self.model.get('page').profile.handle) {
+            var profile = self.model.get('page').profile,
```

Comment:
I don't have it set as a rule in es-lint, but generally variables should be set at the top, which would have avoided the collision here. ES-Lint should have picked up the duplicate variable if this is up to date with master (if it isn't, merge it and re-lint it).